Dockerfile.ci

```diff
@@ -1054,6 +1054,17 @@ function check_download_pendulum() {
     pip check
 }
 
+function uninstall_cassandra_for_python_3_12() {
+    if [[ ${PYTHON_MAJOR_MINOR_VERSION} != "3.12" ]]; then
+        return
+    fi
+    echo
+    echo "${COLOR_BLUE}Uninstalling cassandra in case it was installed via cache${COLOR_RESET}"
```

Comment:
Will this be a permanent thing or shall we have a note with a "open bug ticket" to clean this up?
I assume at some point of time in future cassandra will support 3.12?

Reply:
Actually it's not needed any more - I fixed it by pytest `--exclude` parameters this afternoon. Let me remove it completely